.github/workflows/push-deploy-website.yml

```
@@ -52,6 +51,5 @@
         run: |
           git config --global user.email "actions@github.com"
           git config --global user.name "gh-actions"
-          yarn install --frozen-lockfile
-          yarn deploy
- 
+          npm ci
+          npm run deploy
```

yamllint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)